src/plugins/intel_gpu/include/intel_gpu/runtime/layout.hpp

```diff
@@ -307,7 +311,7 @@ struct layout {
     }
 
     /// Number of elements to be stored in this memory layout
-    size_t count() const;
+    size_t count(bool include_padding = false) const;
```

Comment:
I suggest having a separate method for that. Maybe `padded_count` ?

Reply:
Updated